### DISCUSSION TITLE
Next/image takes seconds to generate images

### DISCUSSION THREAD

**Discussion by etruska** [vercel/next.js#50157] [Help]
### Summary

Hi,
I have a Next.js app that keeps crashing on request timeout from time to time. Strangely enough, it usually starts with _next/image request, which can get from few seconds to even more than 30.
Has someone experienced such extreme loading times?   

Also is somewhere explained how next/image works internally? My assumption is that once you ask for a image url/width/quality combination, it's generated to the cache and returned instantly next time. But I'm not that sure.

I'm using "next": "13.2.4" and "sharp": "^0.31.3".

Any help is appreciated.

<img width="1445" alt="Screenshot 2023-05-22 at 13 50 16" src="https://github.com/vercel/next.js/assets/24705553/757d1125-f286-464b-bdf4-649e1b21a4cb">

**Comment by icyJoseph** [comment]
- https://nextjs.org/docs/pages/api-reference/components/image#caching-behavior

> The expiration (or rather Max Age) is defined by either the [minimumCacheTTL](https://nextjs.org/docs/pages/api-reference/components/image#minimumcachettl) configuration or the upstream image Cache-Control header, whichever is larger.

Could it be that sharp is not being used at all? though, those loading times are quite bad. Maybe there's not enough resources on the server to do the image optimization, in proper time?

**Comment by etruska** [comment]
@icyJoseph Thanks for the api reference link, I totally missed that. I have minimumCacheTTL set very high though.
Right now I'm looking into other server issues, perhaps that's influencing the poor image loading times.
